tests/test_process_video.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from pathlib import Path
+import numpy as np
+
+from mio.process.video import NoisePatchProcessor, FreqencyMaskProcessor, PassThroughProcessor, MinProjSubtractProcessor
+from mio.models.process import DenoiseConfig
+
+class TestVideoProcessors(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = Path("/tmp/test_output")
```

Comment:
not portable across OS, unfortunately, and likely to hit some existing directory this way. this is why we use the `tmp_path` pytest fixture https://docs.pytest.org/en/latest/how-to/tmp_path.html

Reply:
this would also need to get removed on teardown

Reply:
i'm actually just going to rewrite these with pytest

Reply:
Nice thanks!